Kitodo/src/main/java/org/kitodo/services/file/FileService.java

```diff
@@ -637,14 +638,9 @@ public boolean deleteProcessContent(Process process) {
      */
     public URI getSourceDirectory(Process process) {
         URI dir = getProcessSubTypeURI(process, ProcessSubType.IMAGE, null);
-        FilenameFilter filterVerz = new FilenameFilter() {
-            @Override
-            public boolean accept(File dir, String name) {
-                return (name.endsWith("_" + "source"));
-            }
-        };
+        FilenameFilter filterDirectory = new FileEndFilter("_source");
```

Comment:
the variable name does not match the FileEndFilter. It is a filter FOR the directory. if you want to reflect that, then name it filterForDirectory